ci/terraform/shared/redis.tf

```diff
@@ -6,7 +6,7 @@ resource "aws_elasticache_subnet_group" "sessions_store" {
   count = var.use_localstack ? 0 : 1
 
   name       = "${var.environment}-session-store-cache-subnet"
-  subnet_ids = local.private_subnet_ids
+  subnet_ids = [local.private_subnet_ids, local.protected_subnet_ids]
```

Comment:
I don't think we need to place the store in this subnet, just make it accessible from there?